package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "buying-catalogue-components",
-  "version": "4.2.4",
+  "version": "4.2.5",
```

Comment:
I think we should do a minor update rather than patch as we're adding a component. Also, we should update `package-lock` too. Recommend just running `npm version minor --no-git-tag-version`.